pkg/cinder/const.go

```diff
@@ -33,12 +33,14 @@ const (
 
 	// DefaultsConfigFileName -
 	DefaultsConfigFileName = "00-config.conf"
+	// ServiceConfigFileName -
+	ServiceConfigFileName = "01-config.conf"
```

Comment:
It may be a good idea to rename these files at some point, not in this PR, so that the names are self explanatory:

```
00-global-defaults.conf
01-service-defaults.conf
02-global-custom.conf
03-service-custom.conf
04-service-custom-secrets.conf
```

Reply:
I agree the files should be renamed. There was a reason why I originally chose generic names (they don't even mention "cinder"), but time has passed and I think the descriptive names make more sense. I can update this PR next week (I'm on PTO now), or submit another PR later.